catroid/src/main/java/org/catrobat/catroid/common/ActionScheduler.java

```diff
@@ -28,14 +28,14 @@
 import com.badlogic.gdx.utils.Array;
 
 import org.catrobat.catroid.content.Script;
-import org.catrobat.catroid.content.actions.EventSequenceAction;
+import org.catrobat.catroid.content.actions.EventThread;
 
 import java.util.Iterator;
 
 public class ActionScheduler {
 
-	private Array<Action> actionsToBeStarted = new Array<>();
-	private Array<Action> actionsToBeRemoved = new Array<>();
+	private Array<EventThread> threadsToBeStarted = new Array<>();
```

Comment:
Could you please come up with a better name for these  lists? such as threadsInQueue.